code/modules/mob/living/simple_animal/hostile/regalrat.dm

```diff
@@ -31,7 +31,7 @@
 	unique_name = TRUE
 	faction = list("rat")
 	//check if the rat is busy/rummaging through stuff, as to prevent multitasking
-	var/busy = FALSE
+	var/DOING_INTERACTION = FALSE
 	///The spell that the rat uses to generate miasma
```

Comment:
...What? `DOING_INTERACTION` is a macro, search for it in the codebase. This isn't what I meant 😛

Reply:
Huh.